client/state/data-layer/wpcom/me/two-step/application-passwords/schema.js

```diff
@@ -0,0 +1,18 @@
+export default {
+	type: 'object',
+	properties: {
+		application_passwords: {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					generated: { type: 'string' },
+					name: { type: 'string' },
+				},
+				required: [ 'ID', 'generated', 'name' ],
```

Comment:
FYI- `additionalProperties` only affects the immediate object. If we want to exclude here, we'd need to add again.

Reply:
Good catch, I missed to add it there.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/23585/commits/47e2cb1f0e046af5495ebe1df2cbf12ffae78bcb.